packages/components/avatar/styled-avatar.js

```diff
@@ -106,7 +115,7 @@ const NamedAvatar = styled.div`
 
   ${NoUserSelect}
 `;
-NamedAvatar.defaultProps = { theme: Base };
+NamedAvatar.defaultProps = { theme: { ...Base, interfaceDirection: "ltr" } };
```

Comment:
No need to add an expanding parameter to each object if it is not used in the current block

Reply:
Removed unnecessary default prop